edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/CloudConnectionProvider.cs

```diff
@@ -46,7 +47,8 @@ public class CloudConnectionProvider : ICloudConnectionProvider
             TimeSpan operationTimeout,
             bool useServerHeartbeat,
             Option<IWebProxy> proxy,
-            IProductInfoStore productInfoStore)
+            IProductInfoStore productInfoStore,
+            IModelIdStore modelIdStore)
```

Comment:
Wonder if these should be combined into a metadataStore. Check with @ancaantochi if it makes sense.